cmd/mistry/main.go

```diff
@@ -139,6 +128,31 @@ EXAMPLES:
 					Usage:       "Output the build result in JSON format to STDOUT (implies verbose: false)",
 					Destination: &jsonResult,
 				},
+
+				// transport flags
+				cli.BoolFlag{
+					Name:        "async",
```

Comment:
I think the term "async" carries more weight than we want to give to this option. How about something like:

- no-wait
- no-fetch
- no-artifacts
- skip-artifacts
- no-block